src/Utility/Model/VatNumberCheck.php

```diff
@@ -67,18 +68,20 @@ public function check(string $prefixedVatNumber): bool
     /**
      * Returns an initialized Soap data source.
      *
+     * @param array<string, mixed> $options An array of options.
      * @return Soap the soap datasource
      */
-    protected function getSoapDataSource(): Soap
+    protected function getSoapDataSource(array $options = []): Soap
     {
+        $defaultOptions = [
+            'exceptions' => true,
+        ];
+        $options = array_merge($options, $defaultOptions);
```

Comment:
I would say, otherwise there's no way to override `exceptions`:

```php
$options = array_merge($defaultOptions, $options);
```

Reply:
oh wow, that makes way more sense :)